InfoTeamProject21/InfoTeamProject21/Person.vb

```diff
@@ -1,3 +1,73 @@
-﻿Public Class Person
+﻿Option Strict On
+Option Explicit On
+Option Infer Off
+Public Class Person
+    '<<Attributes>>
+    Private _Name As String
+    Private _Age As Integer
+    Private _Diseases As Disease
```

Comment:
@Ruan94bez Please move this to the disease class as -Private _People() as Person